cmd/minikube/cmd/delete_test.go

```diff
@@ -117,7 +117,7 @@ func TestDeleteProfile(t *testing.T) {
 				t.Errorf("Profile folder of profile \"%s\" was not deleted", profile.Name)
 			}
 
-			pathToMachine := cluster.MachinePath(profile.Name, localpath.MiniPath())
+			pathToMachine := machine.Path(profile.Name, localpath.MiniPath())
```

Comment:
I feel like this should be localpath.Machine. Thoughts?

Reply:
done